FBC/ProductsAndServices/FinancialProductsAndServices.rdf

```diff
@@ -985,15 +986,16 @@
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-pas-fpas;TradeIdentifier"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fbc-pas-fpas;hasGeneratingEntityIdentifier"/>
-				<owl:someValuesFrom rdf:resource="&fibo-be-le-lei;LegalEntityIdentifier"/>
+				<owl:onProperty rdf:resource="&fibo-fbc-pas-fpas;hasGeneratingEntity"/>
```

Comment:
we don't a good job of defining "generated". And what's the significance of the entity. If market maker A invokes a service run by provider B using code written by software company C to generate a UTI then which entity is said to have "generated" it?

Reply:
ISO must have an old definition that doesn't account for program trading.  In program trading of course the UTI is programmatically generated.  For a manually entered trade, the trader uses a pre-printed trade ticket that has the UTI pre-assigned.  Thus it is right that the entity assigns the UTI in both cases.  In both cases, each trade is one transaction.  In the case of program trading, multiple trades are created in what's known as a trade list.  But here again, each trade in the trade list corresponds to one transaction.

Reply:
@jfgemski The ISO specification is from 2020, so really recent.  I can add this note, but are you certain that in every case there is a 1-1 correspondence between the definition of what a trade is and what a transaction is?  That wasn't the case at Nordea - they would execute multiple transactions as part of a 'trade', which might include more than one counterparty.  They also had an overarching notion of a 'deal', and there might be additional internal hedge transactions that would be considered part of the deal under an overall identifier for a given trade, and then the various transactions would be linked to that overarching deal/trade.

Reply:
@jfgemski Our definition of trade is: "an agreement between parties participating in a voluntary action of buying and selling goods and services" - very general on purpose, though it came from several sources.  There are a number of notes associated with it as well.

In a subsequent issue, we should probably add the notion of  'program trade', and possibly take another look at the lifecycle definitions in that ontology for the lifecycle of a trade.  I have at least one or two references that may be useful.  But the intent of this issue was to eliminate some schizophrenia with multiple securities transaction identifier / unique transaction identifier concepts that were disconnected.  I think this resolves that issue, even if we determine there is still some ambiguity to resolve.

Reply:
What we'll find is that different entities have different definitions of what they consider to be a trade and a deal.  Nordea's concept of a trade can have multiple transactions.  (I wonder what they think is the difference between a trade and a deal.)  In any case, each transaction has its own UTI.  Let's close this.

Reply:
@rivettp This last statement with respect to individual components of a package (which might reflect a structured product or something like that, at least it did at Nordea), is why internally they needed to track (1) an internal trade identifier, and (2) one or in this case more than one associated transaction-specific identifier.  But that's not inconsistent with a UTI being a subclass of trade identifier due to the restrictions we have now.